bin/lamassu-coinatmradar

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+SEED="$(cat ~/seeds/seed.txt)"
+
+echo
+echo "Here is the 'External ID' of your paired machine(s), for use under the 'ATM / Teller details' of your CoinATMRadar listing:"
+echo
+paste <(su - postgres -c "psql \"lamassu\" -Atc \"select device_id from devices\"" | cut -c -32) <(su - postgres -c "psql \"lamassu\" -Atc \"select name from devices\"") | column -s $'\t' -t
```

Comment:
Shouldn't this be a single JOIN select statement?

Reply:
Good point. This allowed nice formatting of the result, though just pushed a change with similar formatting using a single select statement.